src/components/calcite-input/calcite-input.tsx

```diff
@@ -316,7 +324,9 @@ export class CalciteInput implements LabelableComponent, FormComponent {
 
   componentShouldUpdate(newValue: any, oldValue: any, property: string): boolean {
```

Comment:
while looking at types, `newValue` and `oldValue` both have to be `string` right?